### ISSUE TITLE
Where do we go now?

### ISSUE THREAD

**Issue by kelvindecosta** [acmbpdc/openlib.cs#9]
While migrating from the READMEs to a site, I had a few thoughts:

*   Styling the site would make the library look much better than it does on GitHub

*   Users can still download textbooks.
    The implementation of a script that I had in mind would not provide the control
    a user has when downloading only textbooks of their choice.

The following are some questions I had regarding the advancements of this project :

*   Do we need a section on **Course Prerequisites **?

    The information is useless for people outside of BITS.
    Moreover, before registration, everyone gets a mail that always provides the updated information

    How about we change this section to **Related Courses** instead?

    The problem with this approach is that the courses in our field are quite well connected.
    In other words, almost everything is related. If we can put forward a concise list of courses
    that are related to a course, it might benefit the user's learning experience.

*   Can we get rid of the old master branch?

    The site can be much more user-friendly than the GitHub README.
    We can merge the [`gh-pages`](https://github.com/acmbpdc/openlib.cs/tree/gh-pages) branch into master to do this.
    All we need is a README in [`gh-pages`](https://github.com/acmbpdc/openlib.cs/tree/gh-pages)

    This leads to the question of a separate library on [**GitBook**](https://www.gitbook.com/).
    Honestly, GitBook sucks

**Comment by eash-98** [comment]
* Reg scripts, I'd say let's have them because it's a handy feature for peeps who know what they're doing. Or we can settle for the community to make scripts. 
* I'm **for** prereqs because I think it's a universal thing, a property of the subjects we have..if that tastes better going down.
